package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "displayName": "VR Builder",
+  "name": "co.mindport.vrbuilder.core",
```

Comment:
VR Builder 1.x was named co.mindport.builder.core. Would it make sense to keep the same name? Technically it's the same product, but it spent a couple years as a .unitypackage in the meantime, so it's also fine to brand it as different.

Reply:
After talking about it I'd leave vrbuilder.